fido2/pcsc.py

```diff
@@ -165,7 +165,7 @@ def _chain_apdus(
                 resp += lres
             return resp, sw1, sw2
 
-    def _call_apdu(self, apdu: bytes) -> bytes:
+    def _chained_apdu_exchange(self, apdu: bytes) -> bytes:
```

Comment:
The return type looks incorrect here, which is causing the build to fail.

Reply:
Pushed the changes